### PR TITLE
Updated Architecture and Removed Elasticsearch

### PR DESCRIPTION
**Pull request by ayoneeyee**
## Purpose

This issue is to remove all components ElasticSearch from the quickstart. While there's nothing wrong with the ES implementation here, and while it demonstrates how to have another system plug into a data stream, it's slowing MAPro down. The projects that fork this repository are not in the market for ES. And if they were, the ES service would (hopefully) be a separate project that plugs into any number of data streams from Kafka (instead of DynamoDB Streams). The SES emailer functions are seen as a sufficient example of how to build event driven business logic while being lightweight; these emailers should remain. The ES service is a bigger and more expensive example of this, and isn't worth it for us.

#### Linked Issues to Close

https://github.com/CMSgov/macpro-quickstart-serverless/issues/131

## Approach

All ElasticSearch services, components, and support has been removed.
Some of the files removed include:
services/elasticsearch-auth/package.json
services/elasticsearch-auth/yarn.lock
services/elasticsearch/package.json
services/elasticsearch/yarn.lock
services/stream-functions/package.json
services/stream-functions/yarn.lock

## Learning

Removed elasticsearch from the quickstart serverless repo. Then also removed all references of elasticsearch from other files and directories in the following repo. 
Reviewed logs to make sure there are no error and build completed successfully.
https://github.com/CMSgov/macpro-quickstart-serverless/tree/master/services

## Assorted Notes/Considerations
Make sure to review Github Action logs in details for any issue.

Updated the JIRA Ticket of updates and changes

#### Pull Request Creator Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [ ] Someone has been assigned this PR.
- [ ] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

**Comment by mdial89f** [comment]
@ayoneeyee 
1. Please step through the checklist for PR creator
2. In the linked issue to close, you need to use closing language so the PR is automatically closed.  You listed the Issue number correctly, but you need to put the word "Closes" in front of it.

**Comment by mdial89f** [comment]
> @ayoneeyee
> 
> 1. Please step through the checklist for PR creator
> 2. In the linked issue to close, you need to use closing language so the PR is automatically closed.  You listed the Issue number correctly, but you need to put the word "Closes" in front of it.
> 3. The sytle_checks workflow is failing.  You need to run prettier on your branch and commit/push.
>    `npx prettier --write .`

I've just realized this is not a PR to go into master.  You do not need anyone's approval to merge into a non protected branch.

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 A serverless form submission application built and deployed to AWS with the Serverless Application Framework.
 
+
 ## Architecture
 
 ![Architecture Diagram](./.images/architecture.png?raw=true)
```
Comment (dwhitecl) [inline README.md:8]: its hard to leaving a meaningful comment in a PR like this but I PM'ed @ayoneeyee and asked him to remove the cognito portion of the diagram that was leftover from elasticsearch

**Comment by dwhitecl** [comment]
@ayoneeyee this PR does not have any of the changes for removing elastisticsearch. I think you need to change the target branch to master

**Comment by ayoneeyee** [comment]
closing

**Pull request closed without merging**